charts/common/values.test.yaml

```diff
@@ -0,0 +1,307 @@
+            accessModes: [ "ReadWriteOnce" ]
+            storageClassName: myStorageClass
+            resources:
+              requests:
+                storage: 2Gi
+
+      # volumes is a list of volumes to be created as secret or configMap
+      volumes:
+        # name of the volume
+        - name: generic-name
+        # type can either be "secret", "configMap" or "persistentVolumeClaim"
+          type: "persistentVolumeClaim"
+        # claimNameSuffix has to be the same as .pvcs.[].name
+          claimNameSuffix: "name-of-pvc"
+
+      containers:
+        container-1:
+          image:
+            repository: registry-group.mgmtbi.ch/docker_base/nginx
+            tag: "1.16"
+          resources: {}
+          volumeMounts:
+            # name of the volume, same as in components.controller.volumes.name
+            # path where the volume should be mounted
+            # subPath is an optional path within the volume from which the container's volume should be mounted. Useful for single files instead of directories
+            # readOnly mounted read-only if true, read-write otherwise (false or unspecified). Comment out for using default (false)
+            # clearNamePrefix when true, only value of "name" is used, otherwise (and by default) the templated manifest is prefixed with "library.name".
+            - name: generic-name
+              path: /tmp/test1
+              subPath: test1.yml
+            #  readOnly: true
+            #  clearNamePrefix: false
+      initContainers:
+        initContainer-1:
+          image:
+            repository: registry-group.mgmtbi.ch/docker_base/nginx
+            tag: "1.16"
+          envSecret:
+            foo: bar
+          env:
+            foo: bar
+          resources: {}
+          containerSecurityContext:
+            enabled: true
+            capabilities:
+              # add: []
+              drop:
+              - ALL
+          #  readOnlyRootFilesystem: true
+          #  runAsGroup: 1001
+          #  runAsNonRoot: true
+          #  runAsUser: 1001
+  component3:
+    # start common.service
+    services:
+      # specify the name of the service
+      service1:
+        deploy: true
+    # end common.service
+    # start common.networkpolicy
+    networkpolicy:
+      deploy: true
+    # end common.networkpolicy
+    controller:
+      deploy: true
+      type: "Job"
+      completions: 5
+      completionMode: "Indexed"
+      backoffLimit: 2
+      activeDeadlineSeconds: 120
+      ttlSecondsAfterFinished: 0
+      automountServiceAccountToken: false
+      restartPolicy: "Never"
+
+      # podSecurityContext holds pod-level security attributes and common container settings.
+      # Some fields are also present in container.securityContext. Field values of containerSecurityContext take precedence over field values of podSecurityContext.
+      # ref: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#podsecuritycontext-v1-core
```

Comment:
update to general link: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

Reply:
done